doc/build_apps/install_bin.rst

```diff
@@ -38,9 +38,11 @@ The dependencies required to build and run CCF apps can be conveniently installe
 
         $ cd <ccf_path>/getting_started/setup_vm/
         $ ./run.sh app-dev.yml --extra-vars "platform=virtual" --extra-vars "clang_version=15"
+        
+This will install the latest release of CCF, but a specific release can also be specified with ``--extra-vars "ccf_ver=X.Y.Z"`` if desired.
 
-Install
--------
+Installation from .deb
+----------------------
 
 CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases>`_.
```

Comment:
I think this could use an "Alternatively", or "If you want a .deb" or similar, to clarify that it's a separate guide rather than a followup to the quickstart above. Or perhaps the whole page just needs a "here are several different options for installing CCF" intro?